develop/getting-started/tips-and-tricks.md

```diff
@@ -14,12 +14,12 @@ const choices = [
   {
     name: 'IntelliJ IDEA',
     href: './intellij-idea/tips-and-tricks',
-    image: '/assets/develop/getting-started/intellij/logo.svg',
+    image: '/assets/choices/ide/intellij.svg',
```

Comment:
For all, I would maybe look for monochrome icons in simpleIcons rather than use the color ones, as some color logos, like Apple, don't appear very good on dark mode.

Example:

![Screenshot_20251104-095333.png](https://github.com/user-attachments/assets/30f4bc45-794f-4c1d-acbf-631e148681e4)